docs/en/week08/08-3.md

```diff
@@ -0,0 +1,228 @@
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-3/fig_3.png" height="350px" /><br>
+<b>Fig. 3</b>: Visualizing vector z as bubbles in the latent space
+</center>
+
+*In Figure 3 above, each bubble represents an estimated region of $z$, and the arrows represent how the reconstruction term pushes each estimated value away from the others, which is explained more below.*
+
+If there is overlap between any two estimates of $z$, (visually, if two bubbles overlap) this creates ambiguity for reconstruction because the points in the overlap can be mapped to both original inputs.  Therefore the reconstruction loss will push the points away from one another.
+
+However, if we use just the reconstruction loss, the estimates will continue to be pushed away from each other and the system could blow up.  This is where the penalty term comes in.
+
+**Note:** for binary inputs the reconstruction loss is
+
+$$
+l(x, \hat{x}) = - \sum\limits_{i=1}^n [x_i \text{log}(\hat{x_i}) + (1 - x_i)log(1-\hat{x_i})]
+$$
+
+and for real valued inputs the reconstruction loss is
+
+$$
+l(x, \hat{x}) = \frac{1}{2} ||x - \hat{x} ||^2
+$$
+
+### The Penalty Term
+
+The second term is the relative entropy (a measure of the distance between two distributions) between $z$ which comes from a gaussian with mean $\mathbb{E}(z)$, variance $\mathbb{V}(z)$ and the standard normal distribution. If we expand this second term in the VAE loss function we get:
+
+$$
+\beta l_{\text{KL}}(z,\mathcal{N}(\textbf{0}, \mathbb{I}_d)) = \frac{\beta}{2} \sum\limits_{i=1}^d(\mathbb{V}(z) - \text{log}[\mathbb{V}(z)] - 1 + \mathbb{E}(z)^2)
+$$
+
+Where each expression in the summation has four terms. Below we write out and graph the first three terms in *Figure 4*.
+
+$$
+v_i = \mathbb{V}(z_i) - \text{log}[\mathbb{V}(z_i)] - 1
+$$
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-3/fig_4.png" /><br>
+<b>Fig. 4</b>: Plot showing how relative entropy forces the bubbles to have variance = 1
+</center>
+
+
+So we can see that this expression is minimized when $z_i$ has variance 1.  Therefore our penalty loss will keep the variance of our estimated latent variables at around 1.  Visually, this means our "bubbles" from above will have a radius of around 1.
+
+The last term, $\mathbb{E}(z_i)^2$, minimizes the distance between the $z_i$ and therefore prevents the "exploding" encouraged by the reconstruction term. 
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-3/fig_5.png" height="400px" width="500px"/><br>
+<b>Fig. 5</b>: The "bubble-of-bubble" interpretation of VAE
+</center>
+
+*Figure 5 above shows how VAE loss pushed the estimated latent variables as close together as possible without any overlap while keeping the estimated variance of each point around one.*
+
+**Note:** The $\beta$ in the VAE loss function is a hyperparameter that dictates how to weight the reconstruction and penalty terms.
+
+---
```

Comment:
Remove.

Reply:
Just to confirm, should I remove that note?

Reply:
The `---`.